app/assets/javascripts/govuk_publishing_components/analytics/auto-scroll-tracker.js

```diff
@@ -0,0 +1,184 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function AutoScrollTracker ($module) {
+    this.$module = $module
+    this.SCROLL_TIMEOUT_DELAY = 20
+    this.RESIZE_TIMEOUT_DELAY = 100
+    this.PAGEHEIGHT_TIMEOUT_DELAY = 500
+    this.pageHeight = document.querySelector('body').clientHeight
+    this.trackedNodes = []
+    this.config = {
+      ignoreHeadingsInside: ['.govuk-cookie-banner', '.gem-c-feedback', 'footer'],
```

Comment:
I was a bit torn. I was thinking ahead to allowing this to be an option to be passed to this module, which would allow for greater flexibility in the future, but we don't have a requirement for that now and we shouldn't build things we don't need. So I've changed it as you suggested, although I've left it as an array (of one item) of elements that headings are allowed in, so if we do want to expand it in future it's easier.